pydrive2/fs/spec.py

```diff
@@ -542,6 +542,34 @@ def cp_file(self, lpath, rpath, **kwargs):
             buffer = io.BytesIO(stream.read())
             self.upload_fobj(buffer, rpath)
 
+    def mv(self, path1, path2, recursive=False, maxdepth=None, **kwargs):
+
+        if recursive:
+            raise NotImplementedError("Recursive move is not supported")
+        if maxdepth:
+            raise NotImplementedError("Max depth move is not supported")
+
+        file1_name = posixpath.basename(path1)
+        file2_name = posixpath.basename(path2)
+
+        file1_parent = self._parent(path1)
```

Comment:
Q / check: can `path2` be an absolute one, or is it still within the `self.path` directory. 

Do we want to support a way to move files outside, support absolute paths?